lib/spack/docs/getting_started.rst

```diff
@@ -1422,122 +1420,137 @@ Step 1: Install prerequisites
+
+""""
+NASM
+""""
+
+The Netwide Assembler (NASM) is a x86-64 assembler that some Windows packages
+will use to create binaries and can be found at https://www.nasm.us.
+
+"""""
+CMake
+"""""
+
+While the CMake provided by your Microsoft Visual Studio installation should
+suffice for most packages, we still recommend downloading and installing the
+most recent version of the software at https://cmake.org/download/ in case
+of version restrictions.
 
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Step 2: Install and setup Spack
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 We are now ready to get the Spack environment set up on our machine. We
-begin by using Git to clone the Spack repo, hosted at https://github.com/spack/spack.git
-into a desired directory, for our purposes today, called ``spack_install``.
+begin by creating a top-level directory to do our work in: we will call
+it ``spack_install`` in this tutorial. Inside this directory, use Git to
+clone the Spack repo, hosted at https://github.com/spack/spack.git.
 
 The files and scripts used for Windows installation are on the
-``features/windows-support`` branch; ``cd`` into the repo and use
-``git checkout`` to switch to it.
-
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Step 3: Run and configure Spack
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+features/windows-support branch; ``cd`` into the repo and use 
+``git checkout`` to switch to it. Then navigate to 
+``bin`` and copy the ``scripts`` directory and ``spack_cmd.bat`` up to 
+the top-level ``spack_install`` directory.
 
-To use Spack, run ``bin\spack_cmd.bat`` (you may need to Run as Administrator)
-from the spack source tree root. This will provide a Windows command prompt
-with an environment properly set up with Spack and its prerequisites.
-If you receive a warning message that Python is not in your ``PATH``
-(which may happen if you installed Python from the website and not
-the Windows Store), add the location of the Python executable to your
-``PATH`` now.
-
-To configure Spack, first run the following command inside the Spack console:
+Your file structure should look like this after following the above
+steps:
 
 .. code-block:: console
 
-   spack compiler find
+   spack_install
+       |--------spack
+       |--------scripts
+       |--------spack_cmd.bat
 
-This creates a ``.spack`` directory in our home directory, along with a
-``windows`` subdirectory containing a ``compilers.yaml`` file. On a fresh
-Windows install, the only compiler that should be found is your installation
-of Microsoft Visual Studio.
 
-We need to provide the ``config.yaml`` configuration by ourselves. This goes
-in the ``.spack\windows`` directory in your home directory. Open your text
-editor of choice and enter the following lines for ``config.yaml``:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Step 3: Run and configure Spack
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. code-block:: yaml
+To use Spack, run ``spack_cmd.bat`` (you may need to Run as Administrator).  This will provide
```

Comment:
To use Spack, run ``bin\spack_cmd.bat`` (you may need to Run as Administrator) from the spack source tree root.